sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -263,12 +264,35 @@ export class ApplicationWarningsAPIOutDTO {
   canAcceptAssessment: boolean;
 }
 
+export interface ApplicationSupportingUsersAPIOutDTO {
+  supportingUserId: number;
+  supportingUserType: SupportingUserType;
+}
+
+export class ApplicationSupportingUsersAPIOutDTO {
+  supportingUserId: number;
+  supportingUserType: SupportingUserType;
+}
```

Comment:
Why do we have class and interface of same type?

Reply:
Issue during copy and paste between API/Web DTO. It is fixed now.